Doc/library/annotationlib.rst

```diff
@@ -144,6 +144,17 @@ Classes
 
       The exact values of these strings may change in future versions of Python.
 
+   .. attribute:: VALUE_WITH_FAKE_GLOBALS
+      :value: 4
```

Comment:
`VALUE_WITH_FAKE_GLOBALS` is now 2.  But why publish their values in the first place?

Reply:
A wise man wrote these values in PEP 649 (https://peps.python.org/pep-0649/#overview). They are also relevant if you write an annotate function in C. But yes, we can probably do without them in Python code.